docs2/src/cpgqlref.js

```diff
@@ -166,10 +166,9 @@ export const Steps = {
   [StepKind.Typ]: makeStep('typ', StepFamily.NodeTypeStep, 'Visits TYPE nodes'),
 
   // filter step
-  [StepKind.FilterFilter]: makeParameterizedStep('filter', StepFamily.FilterStep, StepParameterType.StepReturningExpression, ReturnType.Step, 'TODO'),
-  [StepKind.FilterNotFilter]: makeParameterizedStep('filterNot', StepFamily.FilterStep, StepParameterType.StepReturningExpression, ReturnType.Step, 'TODO'),
-  [StepKind.WhereFilter]: makeParameterizedStep('where', StepFamily.FilterStep, StepParameterType.BooleanReturningExpression, ReturnType.Step, 'TODO'),
-  [StepKind.WhereNonEmptyFilter]: makeParameterizedStep('whereNonEmpty', StepFamily.FilterStep, StepParameterType.StepReturningExpression, ReturnType.Step, 'TODO'),
```

Comment:
The `WhereNonEmptyFilter` is found at more places in this file, it would be good to delete them as well; e.g. `~L774`

Reply:
done